core/oauth/domain/metric.go

```diff
@@ -0,0 +1,20 @@
+package domain
+
+import (
+	"fmt"
+
+	"flamingo.me/flamingo/v3/framework/opencensus"
+	"go.opencensus.io/stats"
+	"go.opencensus.io/stats/view"
+)
+
+var (
+	// LoginFailCount counts the failed login attempts
+	LoginFailCount = stats.Int64("flamingo/oauth_login_fail_count", "Count of failed login attempts", stats.UnitDimensionless)
+)
+
+func init() {
+	if err := opencensus.View("flamingo/oauth_login_fail_count", LoginFailCount, view.Count()); err != nil {
```

Comment:
Should't this view only be registered if the config `oauth.metrics.failedLoginCountTracking.enabled` is set to `true`. Otherwise the metric would result in a false-positive zero value, wouldn't it?

Reply:
The view will now only registered if the flag is true.